apps/data-brittler/src/jobs/special_events.rs

```diff
@@ -0,0 +1,53 @@
+use std::sync::Arc;
+
+use shared::database::product::special_event;
+use shared::database::MongoCollection;
+
+use super::{Job, ProcessOutcome};
+use crate::global::Global;
+
+pub struct SpecialEventsJob {
+	global: Arc<Global>,
+}
+
+impl Job for SpecialEventsJob {
+	type T = special_event::SpecialEvent;
+
+	const NAME: &'static str = "special_events";
+
+	async fn new(global: Arc<Global>) -> anyhow::Result<Self> {
+		if global.config().truncate {
+			tracing::info!("dropping special_events collections");
+			special_event::SpecialEvent::collection(global.target_db())
+				.untyped()
+				.drop()
+				.await?;
+			let indexes = special_event::SpecialEvent::indexes();
+			if !indexes.is_empty() {
+				special_event::SpecialEvent::collection(global.target_db())
+					.untyped()
+					.create_indexes(indexes)
+					.await?;
+			}
+		}
+
+		Ok(Self { global })
+	}
+
```

Comment:
Are there any special events that we have to transfer with the brittler?

Reply:
From the existing dataset no.

Reply:
Why does this job exist then?

Reply:
It creates them